apps/pie-docs/src/iconData.json

```diff
@@ -6,15 +6,15 @@
             "icons": [
                 {
                     "name": "over-16-filled",
-                    "displayName": "Over 16"
+                    "displayName": "Over 16 Filled"
```

Comment:
should the display name not have `icon` in the name for clarity?

Reply:
I agree that it would make each icon clearer when viewed in isolation, but in the context of the whole page I think it might make it a bit too busier than necessary.

The current designs don't have it but if the designers think it would make things better then I'd be happy for us to make a ticket to change it.

Reply:
There's also the part where your users will try to copy paste over whatever you've got and not consider the context changing in the way you've described. And this is not something you can educate to a team of 100+ devs.

Reply:
I believe there may have been a large effort to remove 'Icon' from the name a while back @webbertakken. I could be mistaken but I seem to remember this. @ashleynolan would know for sure